recipes/pybroom/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "pybroom" %}
+{% set version = "0.2rc1" %}
+{% set sha256 = "506049ae17c60d60094c44713ccddbc9bf45854708f11281c2ef51f9171dd3c5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/tritemio/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [py2k]
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - pandas
+    - lmfit
+
+test:
+  imports:
+    - pybroom
+
+about:
+  home: http://pybroom.readthedocs.io/
+  license: MIT
```

Comment:
We should be including the license file like so. However, that will require PR ( https://github.com/tritemio/pybroom/pull/4 ) to be merged and used to generate a release. Let's hold on that until we are confident we can get this to build right.

```yaml
  license_file: LICENSE.txt
```